.bazelrc.remote

```diff
@@ -11,7 +11,7 @@ build:rbe --define=EXECUTOR=remote
 build:rbe --experimental_inmemory_dotd_files
 build:rbe --experimental_inmemory_jdeps_files
 build:rbe --remote_timeout=3600
-build:rbe --spawn_strategy=remote,local
+build:rbe --spawn_strategy=remote
```

Comment:
turns out this doesn't fix the problem I thought it would, so I'll revert it. 

For some reason, when I'm running `--config=rbe` anything with tag `exclusive-with-local` still runs in series instead of parallel. I'm not sure why. @shs96c do you know why this may be?

Reply:
(it makes a huge difference time-wise when I remove the `exclusive-with-local` tags and running on rbe.

Reply:
Ah, so this is probably why we were previously skipping them on rbe to avoid the performance hit